can_tools/scrapers/official/SC/sc_demographics.py

```diff
@@ -0,0 +1,83 @@
+from itertools import count
+from tableauscraper import TableauScraper
+import us
+import pandas as pd
+from can_tools.scrapers import variables
+
+from can_tools.scrapers.official.base import StateDashboard
+
+
+class SCVaccineDemographics(StateDashboard):
+    has_location = False
+    source = "https://public.tableau.com/app/profile/population.health.dhec/viz/COVIDVaccineDashboard/RECIPIENTVIEW"
+    source_name = "South Carolina Population Health DHEC"
+    location_type = "county"
+
+    state_fips = us.states.lookup("South Carolina").fips
+    fetch_url = "https://public.tableau.com/views/COVIDVaccineDashboard/RECIPIENTVIEW"
+
+    variables = {"total_vaccinations_initiated": variables.INITIATING_VACCINATIONS_ALL}
+
+    # hard coding these because it was annoying to use getFilter on two different sheets
+    race = [
+        "Asian, American Indian or Alaskan Native, Native Hawaiian or Other Pacific Islander",
+        "Black",
+        "Other",
+        "Unknown",
+        "White",
+    ]
+
+    def fetch(self):
+        engine = TableauScraper()
+        engine.loads(self.fetch_url)
+
+        engine = engine.getWorksheet("Vaccine Map By SC residents PEOPLE")
+        filters = engine.getFilters()
+        counties = [
+            t["values"] for t in filters if t["column"] == "Recipient County for maps"
+        ][0]
+
+        data = []
+        for county in counties:
+            print("parsing", county)
```

Comment:
Same question here but now that I've seen it in 2 scrapers I'm guessing it's intentional. 😁